mysql-test/suite/binlog/t/binlog_half_crash_safe_ddl.test

```diff
@@ -50,7 +50,14 @@
 --source include/have_binlog_format_mixed.inc
 
 --disable_query_log
+--let $SYSSE=`SELECT @@default_dd_system_storage_engine`
+if ($SYSSE == "RocksDB") {
+  SET SESSION binlog_format=ROW;
+}
 call mtr.add_suppression("Could not find a valid tablespace file for");
+if ($SYSSE == "RocksDB") {
```

Comment:
nit: looks like duplicate as line 54-line 56

Reply:
anyway, commit first

Reply:
Will do a follow-up PR to fix